include/picongpu/particles/Particles.tpp

```diff
@@ -376,37 +376,6 @@ namespace picongpu
         ParticlesBaseType::template shiftParticles(mapperFactory, onlyProcessMustShiftSupercells);
     }
 
-    template<typename T_Name, typename T_Flags, typename T_Attributes>
```

Comment:
Why could this be removed?

Reply:
This is moved to another class to resolve include issues.